deployment/test/deploy_to_google.sh

```diff
@@ -6,7 +6,12 @@ cluster_name="codeship-gcloud-test-${date}"
 
 codeship_google authenticate
 
-echo "Starting a small cluster with a single instance"
+# echo "Setting context if cluster exists"
```

Comment:
Does it have any negative effects if we set the context in any case?

Reply:
@mlocher It's commented out, so we're not doing it by default